### PR TITLE
Added missing text domains, translations and corrected acronyms

### PR DESCRIPTION
**Pull request by mavas84**
Added missing text domains, translations, and corrected acronyms

### CODE REVIEW

File: UserExperience_LazyLoad_Page_View.php [hunk 1]
```diff
@@ -60,7 +60,7 @@
 		<th>Google Maps</th>
 		<td>
 			<?php Util_Ui::pro_wrap_maybe_start(); ?>
-			<p class="description w3tc-gopro-excerpt" style="padding-bottom: 10px">Lazy load google map</p>
+			<p class="description w3tc-gopro-excerpt" style="padding-bottom: 10px">( 'Lazy load google map', 'w3-total-cache' )</p>
```
Comment (cssjoe) [inline UserExperience_LazyLoad_Page_View.php:63]: Missing PHP tags, use of translation function, and semi-colon.

File: inc/options/about.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <?php include W3TC_INC_DIR . '/options/common/header.php'; ?>
 
 <div id="about">
-    <p><?php _e( 'User experience is an important aspect of every web site and all web sites can benefit from effective caching and file size reduction. We have applied web site optimization methods typically used with high traffic sites and simplified their implementation. Coupling these methods either <a href="http://memcached.org/" target="_blank">memcached</a> and/or opcode caching and the <acronym title="Content Delivery Network">CDN</acronym> of your choosing to provide the following features and benefits:' ) ?></p>
+    <p><?php _e( 'User experience is an important aspect of every web site and all web sites can benefit from effective caching and file size reduction. We have applied web site optimization methods typically used with high traffic sites and simplified their implementation. Coupling these methods either <a href="http://memcached.org/" target="_blank">memcached</a> and/or opcode caching and the <acronym title="Content Delivery Network">CDN</acronym> of your choosing to provide the following features and benefits:', 'w3-total-cache') ?></p>
```
Comment (cssjoe) [inline inc/options/about.php:11]: Missing semi-colon.
Use `__()` inside of a `sprintf()` with a `// translators: 1:` comment.

File: inc/options/browsercache.php [hunk 3]
```diff
@@ -119,7 +119,7 @@
 					<label><input id="browsercache_nocookies" type="checkbox"
 						<?php Util_Ui::sealing_disabled( 'browsercache.' ) ?>
 						name="nocookies" value="1"<?php checked( $browsercache_nocookies, true ) ?> /> <?php _e( "Don't set cookies for static files", 'w3-total-cache' ) ?></label>
-					<p class="description"><?php _e( 'Removes Set-Cookie header for responses.' ) ?></p>
+					<p class="description"><?php _e( 'Removes Set-Cookie header for responses.', 'w3-total-cache' ) ?></p>
```
Comment (cssjoe) [inline inc/options/browsercache.php:122]: Missing semi-colon.

File: inc/options/general.php [hunk 4]
```diff
@@ -126,7 +126,7 @@
 		'key' => 'minify.configuration_overloaded'
 	) );
 ?>
-		<p><?php w3tc_e( 'minify.general.header', 'Reduce load time by decreasing the size and number of <acronym title="Cascading Style Sheet">CSS</acronym> and <acronym title="JavaScript">JS</acronym> files. Automatically remove unnecessary data from <acronym title="Cascading Style Sheet">CSS</acronym>, <acronym title="JavaScript">JS</acronym>, feed, page and post <acronym title="Hypertext Markup Language">HTML</acronym>.' ) ?></p>
+		<p><?php w3tc_e( 'minify.general.header', 'Reduce load time by decreasing the size and number of <acronym title="Cascading Style Sheet">CSS</acronym> and <acronym title="JavaScript">JS</acronym> files. Automatically remove unnecessary data from <acronym title="Cascading Style Sheet">CSS</acronym>, <acronym title="JavaScript">JS</acronym>, feed, page and post <acronym title="Hypertext Markup Language">HTML</acronym>.', 'w3-total-cache' ) ?></p>
```
Comment (cssjoe) [inline inc/options/general.php:129]: Not a translation function.  https://github.com/W3EDGE/w3-total-cache/blob/2.1.5/w3-total-cache-api.php#L601
Use `__()` inside of a `sprintf()` with a `// translators: 1:` comment.

File: inc/options/general.php [hunk 5]
```diff
@@ -323,7 +323,7 @@
 		<p>
 			<?php
 echo sprintf(
-	w3tc_er( 'reverseproxy.general.header', 'A reverse proxy adds scale to an server by handling requests before WordPress does. Purge settings are set on the <a href="%s">Page Cache settings</a> page and <a href="%s">Browser Cache settings</a> are set on the browser cache settings page.' ),
+	w3tc_er( 'reverseproxy.general.header', 'A reverse proxy adds scale to an server by handling requests before WordPress does. Purge settings are set on the <a href="%s">Page Cache settings</a> page and <a href="%s">Browser Cache settings</a> are set on the browser cache settings page.', 'w3-total-cache' ),
```
Comment (cssjoe) [inline inc/options/general.php:326]: Not a translation function. https://github.com/W3EDGE/w3-total-cache/blob/2.1.5/w3-total-cache-api.php#L601
Use `__()` inside of a `sprintf()` with a `// translators: 1:` comment.

File: inc/options/minify.php [hunk 6]
```diff
@@ -136,7 +136,7 @@
 			<?php
 			Util_Ui::config_item( array(
 				'key' => 'minify.js.enable',
-				'label' => '<acronym title="JavaScript">JS</acronym> minify settings:',
+				'label' => __( '<acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ),
```
Comment (cssjoe) [inline inc/options/minify.php:139]: Do not place markup inside of a translation function.  Use `sprint` with `__()` as a parameter.  Also, include a `// translators: 1:` comment when doing so.

File: inc/options/objectcache.php [hunk 7]
```diff
@@ -72,13 +72,13 @@
 
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?> Enable caching for wp-admin requests</label>
+					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?>( ' Enable caching for wp-admin requests', 'w3-total-cache' )</label>
```
Comment (cssjoe) [inline inc/options/objectcache.php:75]: Not a translation function.  Add PHP tags and a semi-colon too.  Might as well concatenate using ` . `.

File: inc/options/objectcache.php [hunk 8]
```diff
@@ -72,13 +72,13 @@
...
 					<p class="description"><?php _e( 'Enabling this option will increase wp-admin performance, but may cause side-effects', 'w3-total-cache' )?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?> Store transients in database</label>
+					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?>( ' Store transients in database', 'w3-total-cache' )</label>
```
Comment (cssjoe) [inline inc/options/objectcache.php:81]: Not a translation function.  Add PHP tags and a semi-colon too.  Might as well concatenate using ` . `.

**Review by cssjoe — CHANGES_REQUESTED** [review]

File: inc/options/objectcache.php [hunk 9]
```diff
@@ -72,13 +72,13 @@
 
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?>( ' Enable caching for wp-admin requests', 'w3-total-cache' )</label>
+					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?><?php _e( 'Enable caching for wp-admin requests', 'w3-total-cache' ); ?></label>
```
Comment (cssjoe) [inline inc/options/objectcache.php:75]: You can change `?><?php` to `;`.

File: inc/options/objectcache.php [hunk 10]
```diff
@@ -72,13 +72,13 @@
...
 					<p class="description"><?php _e( 'Enabling this option will increase wp-admin performance, but may cause side-effects', 'w3-total-cache' )?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?>( ' Store transients in database', 'w3-total-cache' )</label>
+					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?><?php _e( 'Store transients in database', 'w3-total-cache' ); ?></label>
```
Comment (cssjoe) [inline inc/options/objectcache.php:81]: You can change `?><?php` to `;`.

**Review by cssjoe — CHANGES_REQUESTED** [review]

File: inc/options/objectcache.php (hunk 10 above)
Comment (mavas84) [reply inc/options/objectcache.php]: done

File: inc/options/objectcache.php (hunk 9 above)
Comment (mavas84) [reply inc/options/objectcache.php]: done

File: inc/options/minify.php (hunk 6 above)
Comment (mavas84) [reply inc/options/minify.php]: fixed

File: inc/options/about.php (hunk 2 above)
Comment (mavas84) [reply inc/options/about.php]: Done

File: UserExperience_LazyLoad_Page_View.php (hunk 1 above)
Comment (mavas84) [reply UserExperience_LazyLoad_Page_View.php]: Done

File: inc/options/general.php (hunk 4 above)
Comment (mavas84) [reply inc/options/general.php]: Done

File: inc/options/general.php (hunk 5 above)
Comment (mavas84) [reply inc/options/general.php]: changes reverted

File: inc/options/general.php [hunk 11]
```diff
@@ -126,8 +126,18 @@
 		'key' => 'minify.configuration_overloaded'
 	) );
 ?>
-		<p><?php w3tc_e( 'minify.general.header', 'Reduce load time by decreasing the size and number of <acronym title="Cascading Style Sheet">CSS</acronym> and <acronym title="JavaScript">JS</acronym> files. Automatically remove unnecessary data from <acronym title="Cascading Style Sheet">CSS</acronym>, <acronym title="JavaScript">JS</acronym>, feed, page and post <acronym title="Hypertext Markup Language">HTML</acronym>.', 'w3-total-cache' ) ?></p>
-
+		<p>
+			<?php
+				printf  ( 
+    			// translators: 1: CSS acronym open tag, 2: CSS acronym close tag, 3: JS acronym open tag, 4: JS acronym close tag, 5: HTML acronym open tag, 6: HTML acronym and close tag.
+    			 __( w3tc_e ( 'minify.general.header', 'Reduce load time by decreasing the size and number of %1$sCascading Style Sheet%2$s and  %1$sJavaScript%3$s files. Automatically remove unnecessary data from %1$sCascading Style Sheet%2$s, %1$sJavaScript%3$s, feed, page and post %1$sHypertext Markup Language%4$s.', 'w3-total-cache' )
```
Comment (cssjoe) [inline inc/options/general.php]: `w3tc_e` is a wrapper for `w3tc_er`, which already translates.  You can revert this change.

File: inc/options/minify.php [hunk 12]
```diff
@@ -136,7 +136,11 @@
 			<?php
 			Util_Ui::config_item( array(
 				'key' => 'minify.js.enable',
-				'label' => __( '<acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ),
+				'label' => sprintf(
+				// translators: 1: JS acronym open tag, 2: JS acronym and close tag 
+				__(  '%1$sJavaScript%2$s minify settings:', 'w3-total-cache' ))
```
Comment (cssjoe) [inline inc/options/minify.php:141]: There is an extra space at the beginning.  The close parentheses for `sprintf` is not in the correct place.  There should be a comma at the end of the line and the closing parentheses goes at the end of the line for the acronym closing tag string.

**Review by cssjoe — CHANGES_REQUESTED** [review]

File: Cdn_GeneralPage_View.php [hunk 13]
```diff
@@ -9,7 +9,7 @@
 		'key' => 'cdn.configuration_overloaded'
 	) );
 ?>
-<p><?php w3tc_e( 'cdn.general.header', 'Host static files with your <acronym title="Content Delivery Network">CDN</acronym> to reduce page load time.' ); ?>
+<p><?php w3tc_e( 'cdn.general.header', 'Host static files with your <acronym title="Content Delivery Network">CDN</acronym> to reduce page load time.', 'w3-total-cache' ); ?>
```
Comment (cssjoe) [inline Cdn_GeneralPage_View.php:12]: `w3tc_e` does not have a 3rd parameter.  You added a text domain.

File: inc/options/minify.php (hunk 12 above)
Comment (mavas84) [reply inc/options/minify.php]: Done

File: Cdn_GeneralPage_View.php (hunk 13 above)
Comment (mavas84) [reply Cdn_GeneralPage_View.php]: Done!
